### DISCUSSION TITLE
Add Climate Strike License to list of license options when creating repository

### DISCUSSION THREAD

**Discussion by techieshark** [community/community#20856] [Other Feature Feedback, Questions, & Ideas]
Hi,

When I go to create a new repository, GitHub provides the helpful dropdown that allows me to pick a license:

<img width="520" alt="image" src="https://user-images.githubusercontent.com/1072292/179715170-536a73b0-dd31-4686-87fa-fdee666daac7.png">

This is great. I often use one of these options.

However, I would love it if GitHub also provided the option to select the Climate Strike License.

You can read more about it here -
https://github.com/climate-strike/license

but in a nutshell:

> Climate Strike Software is software that uses the [Climate Strike License](https://github.com/climate-strike/license/blob/master/licenses), a software license that developers can use to prohibit the use of their code by applications or companies that threaten to accelerate climate change through fossil fuel extraction.

The license is still a fairly nascent idea, but I would be very interested to hear from GitHub as to what the process is for adding this or other licenses to that drop down.

Thank you.

cc @frnsys

**Comment by svick** [comment]
Personally, I think that GitHub should not promote the use of non-open source licenses.

**Comment by jfullstackdev** [comment]
you can simply create a new file and paste it
